src/tests/test_bootstrap_in_place.py

```diff
@@ -76,19 +93,22 @@ def installer_gather(self, ip: str, ssh_key: str, out_dir: str):
     # ssl handshake failures at server side are not transient thus we cannot rely on curl retry mechanism by itself
     @JunitTestCase()
     @retry.retry(exceptions=Exception, tries=3, delay=30)
-    def download_live_image(self, download_path: str):
+    def download_live_image(self, download_path: str, ocp_major_minor: str):
         if os.path.exists(download_path):
             log.info("Image %s already exists, skipping download", download_path)
             return
 
+        rhcos_map = RHCOSImages.ver_to_url_map
+        download_image_url = rhcos_map.get(ocp_major_minor, rhcos_map["4.10"])
+
         log.info("Downloading iso to %s", download_path)
-        # TODO: enable fetching the appropriate rhcos image
         utils.run_command(
-            f"curl https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.8/4.8.2/rhcos-live.x86_64.iso"
-            f" --retry 10 --retry-connrefused -o {download_path} --continue-at -"
+            f"curl {download_image_url} --retry 10 --retry-connrefused -o {download_path} --continue-at -"
         )
 
+    # retry to avoid occassional quay hiccups
     @JunitTestCase()
+    @retry.retry(exceptions=Exception, tries=5, delay=30)
```

Comment:
what do you say about applying retries on the specific ``run_container`` action?
seems redundant (and maybe problematic) to redo other unintended failures in this method